sql/test/replication/model.py

```diff
@@ -369,15 +387,18 @@ async def stop(schedule):
       async def push(schedule):
         proc = schedule.getScheduleLocal(key)
 
-        payload = stream.recv(schedule)
-        proc.stdin.write(payload)
-        await proc.stdin.drain()
+        for payload in stream.recv(schedule):
+          proc.stdin.write(payload)
+          await proc.stdin.drain()
 
-        expectAck = payload.decode().startswith("^")
-        while expectAck:
-          ack = await proc.stdout.readline()
-          if ack.decode().startswith(":"):
-            return
+          payload = payload.decode()
+          expectAck = payload.startswith("^")  and not "!rebuild" in payload
```

Comment:
(Just a comment, I know this is reasonable/idiomatic python) but `and not "!rebuild" in payload` syntax is unhinged